docs/campaign/quests/wallet.md

```diff
@@ -2,19 +2,19 @@
 
-- Login to AirLyft and create an event. On the event page, click on select Apps tab where you can find the Wallet block. 
+- Login to AirLyft and create an event. On the event page, click on Quests tab where you can find the Wallet block. 
 
 ![Wallet Main](../../images/WalletMain.png)
 
 - Next, select the type of wallet address you want to collect from users. Currently we provide 2 options which is "EVM wallet address" and "Dotsama Wallet Address", but further development will be done so that projects can create wallet collect tasks for any blockchain. 
 
 ![Wallet Options](../../images/WalletEvmDotsama.png)
 
-- Let's first select EVM Wallet address. Click on it and you will be provided with two input where firstly, you need to enter the basic details of task such as Title, Description, Entries. And the second input is to select the blockchain for which you want to verify and collect users public address. Once selected, click on Add Task to add this to your upcoming campaign.
+- Let's first select EVM Wallet address. Click on it and you will be provided with two input where firstly, you need to enter the basic details of task such as Title, Description, cFuel and XP. And the second input is to select the conditions you want for the task, there are four options: Task Completed (based on another task's status), Date (before or after a set date), Max Participants (limits number of completions). Then, select the blockchain for which you want to verify and collect users public address. Then, select the 'Wallet Collection Type' and 'Verification Type' you want. Once selected, click on Add Task to add this to your upcoming campaign.
 
 ![EVM Wallet Address Collect](../../images/EVMWalletCollect.png)
 
-- Moving towards collection of Dotsama Wallet Address. Click on it and you will be provided with two input where firstly, you need to enter the basic details of task such as Title, Description, Entries. And the second input is to select from the list of Dotsama parachains, sister chains from the list, for which you want to verify and collect users public address. Once selected, click on Add Task to add this to your upcoming campaign.
+- Moving towards collection of Dotsama Wallet Address. Click on it and you will be provided with two input where firstly, you need to enter the basic details of task such as Title, Description, cFuel and XP. And the second input is to select the conditions you want for the task, there are four options: Task Completed (based on another task's status), Date (before or after a set date), Max Participants (limits number of completions). Then, select from the list of Dotsama parachains, sister chains from the list, for which you want to verify and collect users public address. Then, select the 'Wallet Collection Type' and 'Verification Type' you want. After that choose the wallets you want to allow. Once selected, click on Add Task to add this to your upcoming campaign.
```

Comment:
This & the point above have similar content about Title, Description etc...

Reply:
Okay, I'll fix that issue.